iac/terraform/aws/donor_search/stepFunction.tf

```diff
@@ -73,6 +80,7 @@ resource "aws_sfn_state_machine" "donor_search_state_machine" {
 
   definition = templatefile("${path.module}/donor_search.json", {
     DONOR_CALCULATE_LAMBDA_ARN_PLACEHOLDER = module.step_function_lambda["calculate-donors-to-notify"].lambda_arn
+    GEOHASH_CALCULATE_LAMBDA_ARN_PLACEHOLDER = module.step_function_lambda["donor-search-evaluator"].lambda_arn
```

Comment:
Don't use placeholder in param name

Reply:
updated in 0d5f0245edf91b83b3cc1ea2f7c82e86225e14dc